src/main/java/domain/engine/LottoMachine.java

```diff
@@ -0,0 +1,25 @@
+package domain.engine;
+
+import domain.generator.LottoNumberGenerator;
+import domain.model.LottoNumbers;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoMachine {
+    private static final int LOTTO_PRICE = 1000;
+    private final LottoNumberGenerator lottoNumberGenerator;
```

Comment:
```suggestion
    private static final int LOTTO_PRICE = 1000;
    
    private final LottoNumberGenerator lottoNumberGenerator;
```

상수와 멤버변수 간에 개행으로 구분지으면 어떨까요?